docs/docs/deploying-to-heroku.md

```diff
@@ -26,25 +26,41 @@ You can optionally add the buildpacks to `app.json` if you want to take advantag
 }
 ```
 
-Add a `heroku-postbuild` script in your `package.json`:
+Heroku will automatically detect and run the `build` script from your `package.json` which should already look like this:
 
 ```json:title=package.json
 {
   "scripts": {
-    "heroku-postbuild": "gatsby build"
+    "build": "gatsby build"
   }
 }
 ```
 
 Finally, add a `static.json` file in the root of your project to define the directory where your static assets will be. You can check all the options for this file in the [heroku-buildpack-static](https://github.com/heroku/heroku-buildpack-static#configuration) configuration.
 
+The following configuration will give you a good start point in line with Gatsby's [suggested approach to caching](/docs/caching/).
+
 ```json:title=static.json
 {
   "root": "public/",
   "headers": {
-    "/**.js": {
+    "/**/": {
```

Comment:
Only parts in static should be cached with immutable, all other things shouldn't.
```md
    "/**/": {
      "Cache-Control": "public, max-age=0, must-revalidate"
    },
    "/static/**": {
      "Cache-Control": "public, max-age=31536000, immutable"
    },
```

Reply:
Wouldn't this config miss other js/css files in the webroot? I think they too can be cached right? My understanding was It's just the HTML we are wanting to re-validate.
https://www.gatsbyjs.org/docs/caching/#javascript-and-css

Reply:
Oh yeah, you are totally right. That's my mistake, for some reason I thought all js was stored in static. 

Sorry for confusing you and thanks for setting me straight!